plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetFileWriter.java

```diff
@@ -136,6 +149,17 @@ public void commit()
             }
             throw new TrinoException(HIVE_WRITER_CLOSE_ERROR, "Error committing write parquet to Hive", e);
         }
+
+        if (validationInputFactory.isPresent()) {
+            try {
+                long startThreadCpuTime = THREAD_MX_BEAN.getCurrentThreadCpuTime();
+                parquetWriter.validate(validationInputFactory.get());
+                validationCpuNanos += THREAD_MX_BEAN.getCurrentThreadCpuTime() - startThreadCpuTime;
+            }
+            catch (IOException | UncheckedIOException e) {
+                throw new TrinoException(HIVE_WRITE_VALIDATION_FAILED, e);
```

Comment:
should `rollbackAction` be called here?

Reply:
When the commit is unsuccessful due to the exception, abort gets called automatically which will perform the rollback